doc/sfp-cmis/Interface-Link-bring-up-sequence.md

```diff
@@ -184,18 +185,246 @@ if transceiver is not present:
+            XCVRDMT ->> APPL_DB: PORT_TABLE:<lport>.MEDIA_SETTINGS_SYNC_STATUS = MEDIA_SETTINGS_DEFAULT
+        end
+    end
+    Note over APPL_DB: PORT_TABLE:<lport><br>CMIS_REINIT_REQUIRED : true/false<br>MEDIA_NOTIFY_REQUIRED : true/false
+    XCVRDMT ->> CmisManagerTask: Spawns
+    XCVRDMT ->> DomInfoUpdateTask: Spawns
+    XCVRDMT ->> SfpStateUpdateTask: Spawns
+    par XCVRDMT, CmisManagerTask, SfpStateUpdateTask, DomInfoUpdateTask
+        loop Wait for stop_event else poll every 60s
+            DomInfoUpdateTask->>DomInfoUpdateTask: Update TRANSCEIVER_DOM_SENSOR,<br>TRANSCEIVER_STATUS (HW section)<br>TRANSCEIVER_PM tables
+        end
+        loop Wait for stop_event
+            XCVRDMT->>XCVRDMT: Check for changes in PORT_TABLE and act upon receiving DEL event
+        end
+        Note over CmisManagerTask: Subscribe to CONFIG_DB:PORT,<br>STATE_DB:TRANSCEIVER_INFO and STATE_DB:PORT_TABLE
+        loop Wait for stop_event
+            Note over CmisManagerTask: Start the CMIS SM and act based on subscribed DB related changes
+        end
+        Note over SfpStateUpdateTask: _post_port_sfp_info_and_dom_thr_to_db_once<br>_init_port_sfp_status_tbl<br>Subscribe to CONFIG_DB:PORT
+        loop Wait for stop_event
+            SfpStateUpdateTask ->> SfpStateUpdateTask: Handle config change event<br>retry_eeprom_reading()<br>_wrapper_get_transceiver_change_event
+        end
+    end
+```
+
+## SfpStateUpdateTask's role to notify media settings to OA
+
+```mermaid
+sequenceDiagram
+    participant OA
+    participant APPL_DB
+    participant SfpStateUpdateTask
+
+    Note over SfpStateUpdateTask: Subscribe to CONFIG_DB:PORT,<br>STATE_DB:TRANSCEIVER_INFO and STATE_DB:PORT_TABLE
+    Note over SfpStateUpdateTask: Following loop represents _post_port_sfp_info_and_dom_thr_to_db_once
+    loop lport in logical_port_list
+        alt post_port_sfp_info_to_db != SFP_EEPROM_NOT_READY
+             Note over SfpStateUpdateTask: post_port_dom_threshold_info_to_db
+            opt PORT_TABLE:<lport>.MEDIA_SETTINGS_SYNC_STATUS != MEDIA_SETTINGS_DONE
+              opt if lport supports media settings
+                  SfpStateUpdateTask ->> APPL_DB: PORT_TABLE:<lport>.MEDIA_SETTINGS_SYNC_STATUS = MEDIA_SETTINGS_NOTIFIED
+                  APPL_DB -->> OA: Notify media settings for ports
+                  Note over OA: Disable admin status<br>setPortSerdesAttribute
+                  OA ->> APPL_DB: PORT_TABLE:<lport>.MEDIA_SETTINGS_SYNC_STATUS = MEDIA_SETTINGS_DONE
+                  Note over OA: initHostTxReadyState
+                end
+            end
+        else
+            Note over SfpStateUpdateTask: retry_eeprom_set.add(lport)
+        end
+    end
+    Note over SfpStateUpdateTask: _init_port_sfp_status_tbl<br>Subscribe to CONFIG_DB
+    loop Wait for stop_event
+        SfpStateUpdateTask ->> SfpStateUpdateTask: Handle config change event<br>retry_eeprom_reading()<br>_wrapper_get_transceiver_change_event
+    end
+```
+
```

Comment:
When we set the MEDIA_SETTING_NOTIFY I would also mention on the flow that actual SI params from media_setting.json are pushed to PORT TABLE in APP DB. Someone can think that only Media Setting notification is set in DB